Dockerfile

```diff
@@ -0,0 +1,31 @@
+#See https://aka.ms/containerfastmode to understand how Visual Studio uses this Dockerfile to build your images for faster debugging.
+
+FROM mcr.microsoft.com/dotnet/aspnet:5.0 AS base
+WORKDIR /app
+EXPOSE 80
+EXPOSE 443
+
+FROM mcr.microsoft.com/dotnet/sdk:5.0 AS build
+WORKDIR /src
+COPY ["NuGet.Config", "."]
+COPY ["src/Hive/Hive.csproj", "src/Hive/"]
+COPY ["src/Hive.Plugins/Hive.Plugins.csproj", "src/Hive.Plugins/"]
+COPY ["src/Hive.Analyzers/Hive.Analyzers.csproj", "src/Hive.Analyzers/"]
+COPY ["src/CodeGen/Hive.CodeGen/Hive.CodeGen.csproj", "src/CodeGen/Hive.CodeGen/"]
+COPY ["src/CodeGen/Hive.CodeGen.Attributes/Hive.CodeGen.Attributes.csproj", "src/CodeGen/Hive.CodeGen.Attributes/"]
+COPY ["src/Hive.Utilities/Hive.Utilities.csproj", "src/Hive.Utilities/"]
+COPY ["src/Hive.Versioning/Hive.Versioning.csproj", "src/Hive.Versioning/"]
+COPY ["src/Hive.Permissions/Hive.Permissions.csproj", "src/Hive.Permissions/"]
+COPY ["src/Hive.Dependencies/Hive.Dependencies.fsproj", "src/Hive.Dependencies/"]
+RUN dotnet restore "src/Hive/Hive.csproj"
+COPY . .
```

Comment:
Does this not just copy everything in `.`? Did CWD change from `dotnet restore`?

Reply:
I'm not entirely sure. However, [Microsoft recommends](https://docs.microsoft.com/en-us/aspnet/core/host-and-deploy/docker/building-net-docker-images?view=aspnetcore-5.0#the-dockerfile) you restore the the project first and THEN copy everything else over for `Dockerfile`s.